packages/stardog-graphql-language-server/src/GraphQlLanguageServer.ts

```diff
@@ -0,0 +1,243 @@
+import {
+  InitializeResult,
+  TextDocumentChangeEvent,
+  InitializeParams,
+  IConnection,
+  TextDocumentPositionParams,
+  CompletionItem,
+  Range,
+  TextEdit,
+  CompletionItemKind,
+  TextDocument,
+  Diagnostic,
+  DiagnosticSeverity,
+} from 'vscode-languageserver';
+import uniqBy from 'lodash.uniqby';
+import {
+  StandardGraphQlParser,
+  StardogGraphQlParser,
+} from 'millan/dist/standalone/millan.graphql';
+import { autoBindMethods } from 'class-autobind-decorator';
+import {
+  errorMessageProvider,
+  AbstractLanguageServer,
+  CompletionCandidate,
+} from 'stardog-language-utils';
+import { ISemanticError } from 'millan';
+
+const SPARQL_ERROR_PREFIX = 'SPARQL Error: ';
```

Comment:
but this is graphql!

Reply:
See my other comment; Stardog's GraphQL extensions allow embedded SPARQL, which we go ahead and validate.

Reply:
I see. And that is working 💯